Automattic/jetpack
# WordAds: Use WPCOM hosting type for Atomic sites

#### Changes proposed in this Pull Request:

There are some ad server differences between WordAds sites hosted on WPCOM and self-hosted Jetpack with regards to slot type availability — inline and sticky ads being a prime example. This PR provides better continuity for sites that had ads enabled on WPCOM then transitioned to Jetpack.

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Visit ads enabled Jetpack site. The value of `__ATA_PP.ht` should be `2` indicating Jetpack self-hosted
* Visit ads enabled Atomic site. The value of `__ATA_PP.ht` should be `1` indicating WPCOM hosted
